Make Dataset/requirements.txt

```diff
@@ -0,0 +1,12 @@
+torch>=1.1
```

Comment:
Do not create directories (or files) with spaces in names. This naming has a lot of disadvantages. Also make sure your directories naming has uniform style. I see 3 styles: Notebooks, images and script, Make Dataset. I would prefer to use snake_case everywhere